examples/energy-management-app/energy-management-common/energy-management-app.zap

```diff
@@ -3502,7 +3534,7 @@
               "storageOption": "External",
               "singleton": 0,
               "bounded": 0,
-              "defaultValue": "",
+              "defaultValue": null,
```

Comment:
None of the Global Elements aka FeatureMap, AttributeList, EventList, etc. currently have the Nullable quality, the default value should either be 0 or empty list.

Reply:
This is just what ZAP produced when saving and it seems to randomly change between "" or null every few days!

Reply:
Do we currently have tests too verify that all those attributes are populated appropriately by the code since they are external? I understand it is tedious to modify Zap files by hand, but those default values, if not populated appropriately, should be failing upon a type check no?

Reply:
defaultValue is ignored when the Storage is set to External but if this should be empty then ZAP has a bug and I will fix it ASAP

Reply:
@jamesharrow have you personally seen ZAP randomly change values or do you mean when different people use ZAP the values change? This is important because either ZAP has an underlying issue involving saving files (unresolved promise?) OR maybe we have a distribution problem. Maybe people are using different versions of ZAP and saving different ZAP files which in turn overwrite the data? Regardless, in order to get to the bottom of the issue it would be highly productive if a bug was filed with ZAP versions and steps to reproduce.

Thank you for pointing this issue out!

Reply:
We are using the version 2023.12.6.
The CI would have caught the differences if the files in this PR were different than the expected output.

Based on that, it seems we are due for an update of our zap version.